pages/index.js

```diff
@@ -13,69 +24,406 @@ export default function Home() {
+                className="a"
+                style={{ marginRight: '12px' }}
+                href="https://apps.irs.gov/pub/epostcard/cor/812908499_201912_990_2021022417751020.pdf"
+              >
+                2019
+              </a>
+              <a
+                target="_blank"
+                className="a"
+                style={{ marginRight: '12px' }}
+                href="https://apps.irs.gov/pub/epostcard/cor/812908499_201812_990_2020013017086607.pdf"
+              >
+                2018
+              </a>
+              <a
+                target="_blank"
+                className="a"
+                style={{ marginRight: '12px' }}
+                href="https://apps.irs.gov/pub/epostcard/cor/812908499_201712_990EZ_2019112216882661.pdf"
+              >
+                2017
+              </a>
+              <a
+                target="_blank"
+                className="a"
+                style={{ marginRight: '12px' }}
+                href="https://apps.irs.gov/pub/epostcard/cor/812908499_201612_990EZ_2019080116538191.pdf"
+              >
+                2016
+              </a>
+            </Box>
+          </div>
+        </Grid>
+        <Box sx={{ borderBottom: 'rgba(229,231,235,1) solid 2px', mb: 3 }} />
+        <Box sx={{ margin: 'auto', textAlign: 'center', mt: 4 }}>
+          <a href="https://hackclub.com/">
+            <img
+              src="https://assets.hackclub.com/flag-standalone.png"
+              width="100px"
+            />
+          </a>
+        </Box>
+      </Container>
+      <Grid
+        sx={{
+          gridGap: '0px',
+          background: '#000',
+          maxHeight: '100vh',
+          overflow: 'hidden',
+          gridTemplateColumns: '25vh 25vh 25vh 25vh',
+          '@media screen and (orientation: portrait)': {
+            gridTemplateColumns: [
+              '1fr 1fr 1fr!important',
+              '1fr 1fr 1fr 1fr!important'
+            ],
+            gridRow: 1,
+            maxHeight: ['66vw']
+          },
+          '@media screen and (max-width: calc(100vh + 600px))': {
+            gridTemplateColumns:
+              'calc(100vh * 2/9) calc(100vh * 2/9) calc(100vh * 2/9)'
+          }
+        }}
+      >
+        {events.map((event, i) => (
+          <Box
+            as="a"
+            title={event.alt}
+            href={event.bankURL}
+            key={event.bankURL + '-' + i}
+            target="_blank"
+            sx={{
+              height: '25vh',
+              '@media screen and (orientation: portrait)': {
+                height: ['33vw', '25vw']
+              },
+              backgroundImage: `linear-gradient(90deg, rgba(${
+                hexToRgb(event.colour).r
+              },${hexToRgb(event.colour).g},${
+                hexToRgb(event.colour).b
+              },0.5) 0%, rgba(${hexToRgb(event.colour).r},${
+                hexToRgb(event.colour).g
+              },${hexToRgb(event.colour).b},0.5) 100%), url(${event.image})`,
```

Comment:
Thoughts on using next/image instead of making it a background image? I think the images are a bit slow to load on initial load.

Reply:
that's a great idea